app/src/userpages/components/StreamPage/List/index.jsx

```diff
@@ -51,26 +47,341 @@ import Sidebar from '$shared/components/Sidebar'
 import SidebarProvider, { SidebarContext } from '$shared/components/Sidebar/SidebarProvider'
 import ShareSidebar from '$userpages/components/ShareSidebar'
 import { ago } from '$shared/utils/time'
-import { subscribeSnippets } from '$utils/streamSnippets'
-
-import styles from './streamsList.pcss'
-
-const { lg } = breakpoints
+import { MD, LG } from '$shared/utils/styled'
+import useModal from '$shared/hooks/useModal'
+import SnippetDialog from './SnippetDialog'
+
+const DesktopOnlyButton = styled(Button)`
+    && {
+        display: none;
+    }
+
+    @media (min-width: ${LG}px) {
+        && {
+            display: inline-flex;
+        }
+    }
+`
 
 export const CreateStreamButton = () => (
-    <Button
-        className={styles.createStreamButton}
+    <DesktopOnlyButton
         tag={Link}
         to={routes.streams.new()}
     >
         <Translate value="userpages.streams.createStream" />
-    </Button>
+    </DesktopOnlyButton>
 )
 
-const Dialogs = {
-    SHARE: 'share',
-    SNIPPET: 'snippet',
-}
+const StreamTable = styled.div`
+    @media (min-width: ${MD}px) {
+        margin-top: 50px;
+        border-top: 1px solid #CDCDCD;
+    }
+
+    @media (min-width: ${LG}px) {
+        margin-top: 0;
+        margin-bottom: auto;
+        border-top: none;
+    }
+`
+
+const StyledListContainer = styled(ListContainer)`
+    && {
+        padding: 0;
+        margin-bottom: 4em;
+    }
+
+    ${StreamTable}:last-child {
+        margin-bottom: 4rem;
+    }
+
+    @media (min-width: ${MD}px) {
+        && {
+            padding-left: 1.5rem;
+            padding-right: 1.5rem;
+        }
+    }
+
+    @media (min-width: ${LG}px) {
+        && {
+            margin-bottom: 0;
+        }
+    }
+`
+
+const LoadingMoreSpacer = styled.div`
```

Comment:
I now see it's a space "holder". How about we bake that into `LoadMore` via a prop like `preserveSpace` (or similar)? Current way makes us having to customize it for every `LoadMore` instance if we need it.

Reply:
agreed